Livecheckables/popt.rb

```diff
@@ -0,0 +1,5 @@
+class Popt
+  livecheck do
+    url "https://github.com/rpm-software-management/popt.git"
+  end
```

Comment:
```suggestion
  # Until rpm.org releases a new version beyond 1.16, it's unclear whether
  # checking the rpm.org archive index (http://ftp.rpm.org/mirror/popt/) would
  # give us the latest version. As of writing this, there's a 1.18 release
  # candidate at: http://ftp.rpm.org/releases/testing/popt-1.18-rc1.tar.gz
  # Checking the Git repo seems like a safe bet until we see how this plays out.
  livecheck do
    url "https://github.com/rpm-software-management/popt.git"
    regex(/^(?:popt)?-v?(\d+(?:[._-]\d+)+)(?:-release)?$/i)
  end
```

Per our discussion on Slack, we can update the formula to use the GitHub repo as the homepage and a stable archive from rpm.org (http://ftp.rpm.org/mirror/popt/popt-1.16.tar.gz). I'll open a PR in homebrew-brew for these changes shortly.

It's not yet clear what will happen once the next version beyond 1.16 is released (e.g., will it be in /mirror/popt/ or somewhere in /releases/), so it's probably safest to check the GitHub repo for now.

With this in mind, we should add a comment to explain the situation and also use a regex to restrict matching a bit (potentially avoiding unstable releases).

Reply:
Alright 👍🏼